app/lib/submission_builder/state_file.rb

```diff
@@ -0,0 +1,19 @@
+module SubmissionBuilder
+  module StateFile
+    def from_state_code_and_year(state_code, year)
+      unless ::StateFile::StateInformationService::STATES_INFO.key?(state_code)
+        raise StandardError state_code
```

Comment:
this is a good place to use that new error class

Reply:
Done!